Tests/Test_Driver_CANbus.c

```diff
@@ -0,0 +1,106 @@
+#include "Tasks.h"
+#include "CANbus.h"
+#include "BSP_UART.h"
+#include "CANConfig.h"
+
+static OS_TCB Task1_TCB;
+static CPU_STK Task1_Stk[128];
+
+#define STACK_SIZE 128
+
+#define CARCAN_FILTER_SIZE (sizeof carCANFilterList / sizeof(CANId_t))
+
+static void assert(bool);
+
+CANDATA_t dataBuf, resultBuf;
+uint64_t data = 0xdeadbeef12345678;
+
+/*
+ * NOTE: This test must be run with car CAN in loopback mode
+ * TODO: automate this, either with arguments to BSP or #define
+ */
```

Comment:
What does this mean?

Reply:
Every time we want to run CAN in loopback mode we need to go to the BSP layer and change the setting. It might be nice to have a macro to set it, so the test can decide if it wants loopback enabled or not

Reply:
Can you make this an issue ticket?